unit-42-intel-objects-feed/playbooks/playbooks.json

```diff
@@ -0,0 +1,640 @@
+              "arguments": {
+                "name": "Unit 42 Intel Objects Feed",
+                "config": "ba7de328-312c-47e2-a7cc-c6dec0254edc",
+                "params": {
+                  "limit": 10,
+                  "last_pull_time": "2023-10-09T04:59:17.834Z"
+                },
+                "version": "1.0.0",
+                "connector": "unit_42_intel_feed",
+                "operation": "fetch_indicators",
+                "operationTitle": "Fetch Indicators",
+                "pickFromTenant": false,
+                "step_variables": []
+              },
+              "status": null,
+              "top": "200",
+              "left": "240",
+              "stepType": "/api/3/workflow_step_types/0bfed618-0316-11e7-93ae-92361f002671",
+              "group": null,
+              "uuid": "80b2ac54-ad93-4893-aa97-8e72a8919035"
+            },
+            {
+              "@type": "WorkflowStep",
+              "name": "Start",
+              "description": null,
+              "arguments": {
+                "route": "a2345eea-6df0-4b32-8154-f5f982640d8a",
+                "title": "AWS Feed: Get Indicators",
+                "resources": [
+                  "alerts"
+                ],
+                "inputVariables": [],
+                "step_variables": {
+                  "input": {
+                    "params": [],
+                    "records": "{{vars.input.records}}"
+                  }
+                },
+                "executeButtonText": "Execute",
+                "noRecordExecution": true,
+                "singleRecordExecution": false
+              },
+              "status": null,
+              "top": "40",
+              "left": "40",
+              "stepType": "/api/3/workflow_step_types/f414d039-bb0d-4e59-9c39-a8f1e880b18a",
+              "group": null,
+              "uuid": "94d7b644-bc6b-46af-bdc5-b95527dc20ee"
+            }
+          ],
+          "routes": [
+            {
+              "@type": "WorkflowRoute",
+              "name": "Start -> Get Indicatorss",
+              "targetStep": "/api/3/workflow_steps/80b2ac54-ad93-4893-aa97-8e72a8919035",
+              "sourceStep": "/api/3/workflow_steps/94d7b644-bc6b-46af-bdc5-b95527dc20ee",
+              "label": null,
+              "isExecuted": false,
+              "group": null,
+              "uuid": "303ee170-8db4-4f0c-9fd6-36be0c732434"
+            }
+          ],
+          "groups": [],
+          "priority": "/api/3/picklists/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+          "uuid": "8e22a20a-6c81-4fd8-a841-b466f58f7744",
+          "id": 2043,
+          "owners": [],
+          "isPrivate": false,
+          "deletedAt": null,
+          "importedBy": [],
+          "recordTags": [
+            "unit-42-intel-feed",
+            "Unit42",
+            "unit_42_intel_feed"
+          ]
+        }
+      ]
+    }
+  ],
+  "exported_tags": [
+    "dataingestion",
+    "create",
+    "fetch",
+    "unit-42-intel-feed",
```

Comment:
this tag 'unit-42-intel-feed' is included twice can you remove one and validate in UI.